src/functionalTest/java/uk/gov/hmcts/divorce/citizen/CitizenSubmitApplicationTest.java

```diff
@@ -42,4 +43,20 @@ public void shouldPassValidationAndGiveSuccessWhenCaseDataValid() throws IOExcep
             .isEqualTo(json(expectedResponse(RESPONSE)));
     }
 
+    @Test
+    public void shouldPassValidationAndGiveSuccessWhenCaseDataValid2() throws IOException {
```

Comment:
The test name could be changed to shouldPassValidationAndGiveSuccessWhenHelpWithFeesApplied